src/index.html

```diff
@@ -22,6 +22,369 @@
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="u-list">
+          <li class="list-item">
+            <a
+              href="./index.html"
+              class="list-item__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              href="./index.html"
+              class="list-item__link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              href="./index.html"
+              class="list-item__link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              href="./index.html"
+              class="list-item__link"
+              data-qa="nav-hover"
+            >
+              laptops &amp computers
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              href="./index.html"
+              class="list-item__link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              href="./index.html"
+              class="list-item__link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              href="./index.html"
+              class="list-item__link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              href="./index.html"
+              class="list-item__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="card-container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__imac-image"
+            src="./images/imac.jpeg"
+            alt="imac"
+          />
+          <h1 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+          <div class="card__prod-code">Product code: 195434</div>
+          <div class="card__review-block">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-count">Reviews: 5</div>
+          </div>
+          <div class="card__price-block">
+            <div class="card__price-text">Price:</div>
+
+            <div class="card__price-value">$2,199</div>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            BUY
+          </a>
+        </div>
```

Comment:
Don't forget about good code formatting.
Fix this in all code.

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/e73677ba-f6dc-4a01-919a-533dd63284b8)